lib/screens/home/widgets/cocktails_app_item.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:boilerplate/constants/assets.dart';
+import 'package:boilerplate/router/router.gr.dart';
+import 'package:boilerplate/styles/colors.dart';
+import 'package:boilerplate/styles/fonts.dart';
+import 'package:boilerplate/styles/sizes.dart';
+import 'package:boilerplate/styles/spacings.dart';
+
+class CocktailsAppItem extends StatelessWidget
+{
+  const CocktailsAppItem({Key? key, required this.itemTitle}) : super(key: key);
+
+  final String itemTitle;
+
+  @override
+  Widget build(BuildContext context)
+  {
+    void routeToItemPage()
+    {
+      context.router.push(CocktailsListRoute(categoryName: itemTitle.toLowerCase().replaceAll(" ", "-")));
+    }
+    return GestureDetector(
+      onTap: ()
+      {
+        routeToItemPage();
+      },
```

Comment:
Let make this as an arrow function to let the code be less verbose 
```suggestion
      onTap: () => routeToItemPage(),
```